parking_lot/shr_careplan.txt

```diff
@@ -15,7 +15,7 @@ EntryElement:	Goal  // Target (Future) Health State
 Based on:		Entity
 Concept:		MTH#C0018017
 Description:	"A broad statement about what should be accomplished for or by a patient or other subject. It represents the intended long-term outcome toward which an individual or a group aspires, and toward which, effort is directed, in the form of activities."
-1..1			Subject
+1..1			Patient
```

Comment:
@markkramerus - should this be revisited?  in the FHIR R4 spec, CareTeam.subject references either a Patient or a Group.

Reply:
Yes, this was on my list to fix.  I have now revisited it. It is now SubjectOfRecord instead of Patient. And carried throughout.